darts/dataprocessing/dtw/_plot.py

```diff
@@ -0,0 +1,137 @@
+from typing import Union, Tuple
+
+import numpy as np
+import xarray as xr
+from matplotlib import pyplot as plt
+from matplotlib import transforms
+
+from ...logging import raise_if_not
+
+
+def plot(self, show_series: bool = False, show_cost: bool = True):
+    """
+    Plot the warp path.
+
+    Parameters
+    ----------
+    show_series:
+        Boolean indicating whether to plot the two time series.
+        Series1 will be plotted below the cost matrix and series2 to the left of the cost matrix.
+
+    show_cost:
+        Boolean indicating whether to plot the cost matrix.
+        Useful for visualizing the effect of the window function.
+
+    Returns
```

Comment:
No need to have an empty returns section here

Reply:
Congratulations